description,command,status,uid,diagnostic,objective
Pass the linting checks run by flake8,cd fibonaccicreator && poetry run task flake8,true,2zdwxa4w1RzMTPppmcTgrA==,,lint
Pass all of the Pytest test cases,cd fibonaccicreator && poetry run task test,false,b6vKWo8XetQV73efl+7ZIA==,"============================= test session starts ==============================	     platform linux -- Python 3.10.12, pytest-7.1.3, pluggy-1.0.0	     rootdir: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator	     collected 0 items / 1 error	     	     ==================================== ERRORS ====================================	     ___________________ ERROR collecting tests/test_fibonacci.py ___________________	     tests/test_fibonacci.py:13: in <module>	         from fibonaccicreator import fibonacci	     fibonaccicreator/fibonacci.py:13: in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     E   NameError: name 'List' is not defined	     =========================== short test summary info ============================	     ERROR tests/test_fibonacci.py - NameError: name 'List' is not defined	     !!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!	     !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!	     =============================== 1 error in 0.47s ===============================",test
Pass the linting checks run by pydocstyle,cd contactsearcher && poetry run task pydocstyle,true,cZdEgIWQoeIofDdrYH/Afg==,,
Run the fibonaccicreator program and ensure that it does not crash with iterative tuple,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativetuple,false,p5K7+j/WnpdDSeiVoM6iFA==,"Traceback (most recent call last):	       File ""<string>"", line 1, in <module>	       File ""/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/importlib/__init__.py"", line 126, in import_module	         return _bootstrap._gcd_import(name[level:], package, level)	       File ""<frozen importlib._bootstrap>"", line 1050, in _gcd_import	       File ""<frozen importlib._bootstrap>"", line 1027, in _find_and_load	       File ""<frozen importlib._bootstrap>"", line 1006, in _find_and_load_unlocked	       File ""<frozen importlib._bootstrap>"", line 688, in _load_unlocked	       File ""<frozen importlib._bootstrap_external>"", line 883, in exec_module	       File ""<frozen importlib._bootstrap>"", line 241, in _call_with_frames_removed	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py"", line 22, in <module>	         from fibonaccicreator import fibonacci	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py"", line 13, in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     NameError: name 'List' is not defined. Did you mean: 'list'?",
Pass the linting checks run by pymarkdown,cd fibonaccicreator && poetry run task markdownlint,true,wC69ew7NwVXVb1dj2hr23w==,,lint
Pass the linting checks run by flake8,cd fibonaccicreator && poetry run task flake8,false,p5K7+j/WnpdDSeiVoM6iFA==,fibonaccicreator/fibonacci.py:13:45: F821 undefined name 'List'	     fibonaccicreator/fibonacci.py:23:46: F821 undefined name 'Tuple'	     fibonaccicreator/fibonacci.py:35:46: F821 undefined name 'Tuple'	     fibonaccicreator/fibonacci.py:50:45: F821 undefined name 'List',lint
Pass the linting checks run by pymarkdown,cd fibonaccicreator && poetry run task markdownlint,true,33x/IdA+veSNbwwV+sZAeg==,,lint
Pass the source code formatting checks run by black,cd fibonaccicreator && poetry run task black,false,b6vKWo8XetQV73efl+7ZIA==,"would reformat /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py	     	     Oh no! ðŸ’¥ ðŸ’” ðŸ’¥	     1 file would be reformatted, 4 files would be left unchanged.",lint
Pass the source code formatting checks run by black,cd fibonaccicreator && poetry run task black,true,2zdwxa4w1RzMTPppmcTgrA==,,lint
Pass all of the Pytest test cases,cd fibonaccicreator && poetry run task test,false,wC69ew7NwVXVb1dj2hr23w==,"============================= test session starts ==============================	     platform linux -- Python 3.10.12, pytest-7.1.3, pluggy-1.0.0	     rootdir: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator	     collected 0 items / 1 error	     	     ==================================== ERRORS ====================================	     ___________________ ERROR collecting tests/test_fibonacci.py ___________________	     tests/test_fibonacci.py:13: in <module>	         from fibonaccicreator import fibonacci	     fibonaccicreator/fibonacci.py:13: in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     E   NameError: name 'List' is not defined	     =========================== short test summary info ============================	     ERROR tests/test_fibonacci.py - NameError: name 'List' is not defined	     !!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!	     !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!	     =============================== 1 error in 0.62s ===============================",test
Pass the linting checks run by flake8,cd fibonaccicreator && poetry run task flake8,false,b6vKWo8XetQV73efl+7ZIA==,fibonaccicreator/fibonacci.py:13:45: F821 undefined name 'List'	     fibonaccicreator/fibonacci.py:23:46: F821 undefined name 'Tuple'	     fibonaccicreator/fibonacci.py:35:46: F821 undefined name 'Tuple'	     fibonaccicreator/fibonacci.py:50:45: F821 undefined name 'List',lint
Pass the linting checks run by pylint,cd fibonaccicreator && poetry run task pylint,true,Rc9Hc3YcbAwPRaxiorY2XA==,,lint
Pass the linting checks run by flake8,cd fibonaccicreator && poetry run task flake8,false,33x/IdA+veSNbwwV+sZAeg==,fibonaccicreator/fibonacci.py:13:45: F821 undefined name 'List'	     fibonaccicreator/fibonacci.py:23:46: F821 undefined name 'Tuple'	     fibonaccicreator/fibonacci.py:35:46: F821 undefined name 'Tuple'	     fibonaccicreator/fibonacci.py:50:45: F821 undefined name 'List',lint
Pass the source code formatting checks run by black,cd fibonaccicreator && poetry run task black,false,33x/IdA+veSNbwwV+sZAeg==,"would reformat /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py	     	     Oh no! ðŸ’¥ ðŸ’” ðŸ’¥	     1 file would be reformatted, 4 files would be left unchanged.",lint
Pass the import checks run by isort,cd fibonaccicreator && poetry run task isort,false,Rc9Hc3YcbAwPRaxiorY2XA==,ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py Imports are incorrectly sorted and/or formatted.	     ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py Imports are incorrectly sorted and/or formatted.,lint
Pass the source code formatting checks run by black,cd contactsearcher && poetry run task black,true,cZdEgIWQoeIofDdrYH/Afg==,,
Run the fibonaccicreator program and ensure that it does not crash with iterative tuple,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativetuple,true,Rc9Hc3YcbAwPRaxiorY2XA==,,
Pass the linting checks run by flake8,cd fibonaccicreator && poetry run task flake8,false,wC69ew7NwVXVb1dj2hr23w==,fibonaccicreator/fibonacci.py:13:45: F821 undefined name 'List'	     fibonaccicreator/fibonacci.py:23:46: F821 undefined name 'Tuple'	     fibonaccicreator/fibonacci.py:35:46: F821 undefined name 'Tuple'	     fibonaccicreator/fibonacci.py:50:45: F821 undefined name 'List',lint
Pass the linting checks run by flake8,cd contactsearcher && poetry run task flake8,false,cZdEgIWQoeIofDdrYH/Afg==,contactsearcher/main.py:8:2: F821 undefined name 'cli'	     contactsearcher/main.py:10:28: F821 undefined name 'typer'	     contactsearcher/main.py:11:20: F821 undefined name 'Optional'	     contactsearcher/main.py:11:29: F821 undefined name 'Path'	     contactsearcher/main.py:11:37: F821 undefined name 'typer'	     contactsearcher/main.py:19:9: F821 undefined name 'typer'	     contactsearcher/main.py:20:15: F821 undefined name 'typer'	     contactsearcher/main.py:25:9: F821 undefined name 'typer'	     contactsearcher/main.py:30:9: F821 undefined name 'typer'	     contactsearcher/main.py:32:5: F821 undefined name 'typer'	     contactsearcher/main.py:33:5: F821 undefined name 'typer'	     contactsearcher/search.py:5:1: F401 'csv' imported but unused,
Pass the linting checks run by pymarkdown,cd fibonaccicreator && poetry run task markdownlint,true,p5K7+j/WnpdDSeiVoM6iFA==,,lint
Pass the linting checks run by pylint,cd fibonaccicreator && poetry run task pylint,false,wC69ew7NwVXVb1dj2hr23w==,************* Module fibonaccicreator.fibonacci	     fibonaccicreator/fibonacci.py:13:28: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:23:29: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:35:29: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:50:28: W0613: Unused argument 'number' (unused-argument)	     	     -----------------------------------	     Your code has been rated at 9.56/10,lint
Pass the import checks run by isort,cd fibonaccicreator && poetry run task isort,false,wC69ew7NwVXVb1dj2hr23w==,ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/tests/test_fibonacci.py Imports are incorrectly sorted and/or formatted.,lint
Pass the linting checks run by pydocstyle,cd fibonaccicreator && poetry run task pydocstyle,true,Rc9Hc3YcbAwPRaxiorY2XA==,,lint
Pass all of the Pytest test cases,cd fibonaccicreator && poetry run task test,false,p5K7+j/WnpdDSeiVoM6iFA==,"============================= test session starts ==============================	     platform linux -- Python 3.10.12, pytest-7.1.3, pluggy-1.0.0	     rootdir: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator	     collected 0 items / 1 error	     	     ==================================== ERRORS ====================================	     ___________________ ERROR collecting tests/test_fibonacci.py ___________________	     tests/test_fibonacci.py:13: in <module>	         from fibonaccicreator import fibonacci	     fibonaccicreator/fibonacci.py:13: in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     E   NameError: name 'List' is not defined	     =========================== short test summary info ============================	     ERROR tests/test_fibonacci.py - NameError: name 'List' is not defined	     !!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!	     !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!	     =============================== 1 error in 0.44s ===============================",test
Run the fibonaccicreator program and ensure that it does not crash with iterative list,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativelist,true,2zdwxa4w1RzMTPppmcTgrA==,,
Pass the linting checks run by pydocstyle,cd fibonaccicreator && poetry run task pydocstyle,true,b6vKWo8XetQV73efl+7ZIA==,,lint
Pass the source code formatting checks run by black,cd fibonaccicreator && poetry run task black,true,wC69ew7NwVXVb1dj2hr23w==,,lint
Pass the import checks run by isort,cd fibonaccicreator && poetry run task isort,false,2zdwxa4w1RzMTPppmcTgrA==,ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/tests/test_fibonacci.py Imports are incorrectly sorted and/or formatted.,lint
Run the fibonaccicreator program and ensure that it does not crash with iterative tuple,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativetuple,false,33x/IdA+veSNbwwV+sZAeg==,"Traceback (most recent call last):	       File ""<string>"", line 1, in <module>	       File ""/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/importlib/__init__.py"", line 126, in import_module	         return _bootstrap._gcd_import(name[level:], package, level)	       File ""<frozen importlib._bootstrap>"", line 1050, in _gcd_import	       File ""<frozen importlib._bootstrap>"", line 1027, in _find_and_load	       File ""<frozen importlib._bootstrap>"", line 1006, in _find_and_load_unlocked	       File ""<frozen importlib._bootstrap>"", line 688, in _load_unlocked	       File ""<frozen importlib._bootstrap_external>"", line 883, in exec_module	       File ""<frozen importlib._bootstrap>"", line 241, in _call_with_frames_removed	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py"", line 22, in <module>	         from fibonaccicreator import fibonacci	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py"", line 13, in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     NameError: name 'List' is not defined. Did you mean: 'list'?",
Pass the linting checks run by mypy,cd fibonaccicreator && poetry run task mypy,true,2zdwxa4w1RzMTPppmcTgrA==,,lint
Pass the linting checks run by pymarkdown,cd fibonaccicreator && poetry run task markdownlint,true,Rc9Hc3YcbAwPRaxiorY2XA==,,lint
Pass all of the Pytest test cases,cd fibonaccicreator && poetry run task test,true,2zdwxa4w1RzMTPppmcTgrA==,,test
Run the fibonaccicreator program and ensure that it does not crash with iterative tuple,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativetuple,false,b6vKWo8XetQV73efl+7ZIA==,"Traceback (most recent call last):	       File ""<string>"", line 1, in <module>	       File ""/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/importlib/__init__.py"", line 126, in import_module	         return _bootstrap._gcd_import(name[level:], package, level)	       File ""<frozen importlib._bootstrap>"", line 1050, in _gcd_import	       File ""<frozen importlib._bootstrap>"", line 1027, in _find_and_load	       File ""<frozen importlib._bootstrap>"", line 1006, in _find_and_load_unlocked	       File ""<frozen importlib._bootstrap>"", line 688, in _load_unlocked	       File ""<frozen importlib._bootstrap_external>"", line 883, in exec_module	       File ""<frozen importlib._bootstrap>"", line 241, in _call_with_frames_removed	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py"", line 22, in <module>	         from fibonaccicreator import fibonacci	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py"", line 13, in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     NameError: name 'List' is not defined. Did you mean: 'list'?",
Pass the import checks run by isort,cd contactsearcher && poetry run task isort,false,cZdEgIWQoeIofDdrYH/Afg==,ERROR: /home/runner/work/content-search-Yanqiao4396/content-search-Yanqiao4396/contactsearcher/contactsearcher/search.py Imports are incorrectly sorted and/or formatted.,
Pass the linting checks run by pydocstyle,cd fibonaccicreator && poetry run task pydocstyle,true,33x/IdA+veSNbwwV+sZAeg==,,lint
Run the fibonaccicreator program and ensure that it does not crash with iterative tuple,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativetuple,true,2zdwxa4w1RzMTPppmcTgrA==,,
Pass the source code formatting checks run by black,cd fibonaccicreator && poetry run task black,false,p5K7+j/WnpdDSeiVoM6iFA==,"would reformat /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py	     	     Oh no! ðŸ’¥ ðŸ’” ðŸ’¥	     1 file would be reformatted, 4 files would be left unchanged.",lint
Pass the linting checks run by pymarkdown,cd fibonaccicreator && poetry run task markdownlint,true,b6vKWo8XetQV73efl+7ZIA==,,lint
Pass the linting checks run by pydocstyle,cd fibonaccicreator && poetry run task pydocstyle,true,2zdwxa4w1RzMTPppmcTgrA==,,lint
Pass the linting checks run by mypy,cd fibonaccicreator && poetry run task mypy,false,wC69ew7NwVXVb1dj2hr23w==,"fibonaccicreator/fibonacci.py:13: error: Name ""List"" is not defined	     fibonaccicreator/fibonacci.py:13: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import List"")	     fibonaccicreator/fibonacci.py:23: error: Name ""Tuple"" is not defined	     fibonaccicreator/fibonacci.py:23: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Tuple"")	     fibonaccicreator/fibonacci.py:35: error: Name ""Tuple"" is not defined	     fibonaccicreator/fibonacci.py:35: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Tuple"")	     fibonaccicreator/fibonacci.py:50: error: Name ""List"" is not defined	     fibonaccicreator/fibonacci.py:50: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import List"")	     Found 4 errors in 1 file (checked 3 source files)",lint
Pass the linting checks run by mypy,cd fibonaccicreator && poetry run task mypy,true,Rc9Hc3YcbAwPRaxiorY2XA==,,lint
Pass the linting checks run by pydocstyle,cd fibonaccicreator && poetry run task pydocstyle,true,wC69ew7NwVXVb1dj2hr23w==,,lint
Pass the import checks run by isort,cd fibonaccicreator && poetry run task isort,false,p5K7+j/WnpdDSeiVoM6iFA==,ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py Imports are incorrectly sorted and/or formatted.	     ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/tests/test_fibonacci.py Imports are incorrectly sorted and/or formatted.,lint
Pass all of the Pytest test cases,cd fibonaccicreator && poetry run task test,false,33x/IdA+veSNbwwV+sZAeg==,"============================= test session starts ==============================	     platform linux -- Python 3.10.12, pytest-7.1.3, pluggy-1.0.0	     rootdir: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator	     collected 0 items / 1 error	     	     ==================================== ERRORS ====================================	     ___________________ ERROR collecting tests/test_fibonacci.py ___________________	     tests/test_fibonacci.py:13: in <module>	         from fibonaccicreator import fibonacci	     fibonaccicreator/fibonacci.py:13: in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     E   NameError: name 'List' is not defined	     =========================== short test summary info ============================	     ERROR tests/test_fibonacci.py - NameError: name 'List' is not defined	     !!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!	     !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!	     =============================== 1 error in 0.59s ===============================",test
Run the fibonaccicreator program and ensure that it does not crash with iterative list,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativelist,false,33x/IdA+veSNbwwV+sZAeg==,"Traceback (most recent call last):	       File ""<string>"", line 1, in <module>	       File ""/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/importlib/__init__.py"", line 126, in import_module	         return _bootstrap._gcd_import(name[level:], package, level)	       File ""<frozen importlib._bootstrap>"", line 1050, in _gcd_import	       File ""<frozen importlib._bootstrap>"", line 1027, in _find_and_load	       File ""<frozen importlib._bootstrap>"", line 1006, in _find_and_load_unlocked	       File ""<frozen importlib._bootstrap>"", line 688, in _load_unlocked	       File ""<frozen importlib._bootstrap_external>"", line 883, in exec_module	       File ""<frozen importlib._bootstrap>"", line 241, in _call_with_frames_removed	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py"", line 22, in <module>	         from fibonaccicreator import fibonacci	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py"", line 13, in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     NameError: name 'List' is not defined. Did you mean: 'list'?",
Run the fibonaccicreator program and ensure that it does not crash with iterative list,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativelist,false,p5K7+j/WnpdDSeiVoM6iFA==,"Traceback (most recent call last):	       File ""<string>"", line 1, in <module>	       File ""/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/importlib/__init__.py"", line 126, in import_module	         return _bootstrap._gcd_import(name[level:], package, level)	       File ""<frozen importlib._bootstrap>"", line 1050, in _gcd_import	       File ""<frozen importlib._bootstrap>"", line 1027, in _find_and_load	       File ""<frozen importlib._bootstrap>"", line 1006, in _find_and_load_unlocked	       File ""<frozen importlib._bootstrap>"", line 688, in _load_unlocked	       File ""<frozen importlib._bootstrap_external>"", line 883, in exec_module	       File ""<frozen importlib._bootstrap>"", line 241, in _call_with_frames_removed	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py"", line 22, in <module>	         from fibonaccicreator import fibonacci	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py"", line 13, in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     NameError: name 'List' is not defined. Did you mean: 'list'?",
Pass the linting checks run by pylint,cd fibonaccicreator && poetry run task pylint,false,p5K7+j/WnpdDSeiVoM6iFA==,************* Module fibonaccicreator.fibonacci	     fibonaccicreator/fibonacci.py:13:28: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:23:29: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:35:29: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:50:28: W0613: Unused argument 'number' (unused-argument)	     	     -----------------------------------	     Your code has been rated at 9.55/10,lint
Pass the linting checks run by mypy,cd fibonaccicreator && poetry run task mypy,false,33x/IdA+veSNbwwV+sZAeg==,"fibonaccicreator/fibonacci.py:13: error: Name ""List"" is not defined	     fibonaccicreator/fibonacci.py:13: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import List"")	     fibonaccicreator/fibonacci.py:23: error: Name ""Tuple"" is not defined	     fibonaccicreator/fibonacci.py:23: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Tuple"")	     fibonaccicreator/fibonacci.py:35: error: Name ""Tuple"" is not defined	     fibonaccicreator/fibonacci.py:35: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Tuple"")	     fibonaccicreator/fibonacci.py:50: error: Name ""List"" is not defined	     fibonaccicreator/fibonacci.py:50: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import List"")	     Found 4 errors in 1 file (checked 3 source files)",lint
Pass the linting checks run by pylint,cd contactsearcher && poetry run task pylint,false,cZdEgIWQoeIofDdrYH/Afg==,************* Module contactsearcher.search	     contactsearcher/search.py:8:31: W0613: Unused argument 'job_description' (unused-argument)	     contactsearcher/search.py:8:53: W0613: Unused argument 'contacts' (unused-argument)	     	     -----------------------------------	     Your code has been rated at 9.55/10,
Pass the linting checks run by pylint,cd fibonaccicreator && poetry run task pylint,false,b6vKWo8XetQV73efl+7ZIA==,************* Module fibonaccicreator.fibonacci	     fibonaccicreator/fibonacci.py:13:28: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:23:29: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:35:29: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:50:28: W0613: Unused argument 'number' (unused-argument)	     	     -----------------------------------	     Your code has been rated at 9.55/10,lint
Pass the linting checks run by mypy,cd fibonaccicreator && poetry run task mypy,false,p5K7+j/WnpdDSeiVoM6iFA==,"fibonaccicreator/fibonacci.py:13: error: Name ""List"" is not defined	     fibonaccicreator/fibonacci.py:13: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import List"")	     fibonaccicreator/fibonacci.py:23: error: Name ""Tuple"" is not defined	     fibonaccicreator/fibonacci.py:23: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Tuple"")	     fibonaccicreator/fibonacci.py:35: error: Name ""Tuple"" is not defined	     fibonaccicreator/fibonacci.py:35: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Tuple"")	     fibonaccicreator/fibonacci.py:50: error: Name ""List"" is not defined	     fibonaccicreator/fibonacci.py:50: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import List"")	     Found 4 errors in 1 file (checked 3 source files)",lint
Pass the linting checks run by pylint,cd fibonaccicreator && poetry run task pylint,false,33x/IdA+veSNbwwV+sZAeg==,************* Module fibonaccicreator.fibonacci	     fibonaccicreator/fibonacci.py:13:28: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:23:29: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:35:29: W0613: Unused argument 'number' (unused-argument)	     fibonaccicreator/fibonacci.py:50:28: W0613: Unused argument 'number' (unused-argument)	     	     -----------------------------------	     Your code has been rated at 9.55/10,lint
Pass the linting checks run by pymarkdown,cd contactsearcher && poetry run task markdownlint,false,cZdEgIWQoeIofDdrYH/Afg==,"../README.md:2:1: MD012: Multiple consecutive blank lines [Expected: 1, Actual: 2] (no-multiple-blanks)",
Pass the linting checks run by pymarkdown,cd fibonaccicreator && poetry run task markdownlint,true,2zdwxa4w1RzMTPppmcTgrA==,,lint
Pass all of the Pytest test cases,cd fibonaccicreator && poetry run task test,true,Rc9Hc3YcbAwPRaxiorY2XA==,,test
Run the fibonaccicreator program and ensure that it does not crash with iterative tuple,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativetuple,false,wC69ew7NwVXVb1dj2hr23w==,"Traceback (most recent call last):	       File ""<string>"", line 1, in <module>	       File ""/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/importlib/__init__.py"", line 126, in import_module	         return _bootstrap._gcd_import(name[level:], package, level)	       File ""<frozen importlib._bootstrap>"", line 1050, in _gcd_import	       File ""<frozen importlib._bootstrap>"", line 1027, in _find_and_load	       File ""<frozen importlib._bootstrap>"", line 1006, in _find_and_load_unlocked	       File ""<frozen importlib._bootstrap>"", line 688, in _load_unlocked	       File ""<frozen importlib._bootstrap_external>"", line 883, in exec_module	       File ""<frozen importlib._bootstrap>"", line 241, in _call_with_frames_removed	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py"", line 13, in <module>	         from fibonaccicreator import fibonacci	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py"", line 13, in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     NameError: name 'List' is not defined. Did you mean: 'list'?",
Pass the linting checks run by pylint,cd fibonaccicreator && poetry run task pylint,true,2zdwxa4w1RzMTPppmcTgrA==,,lint
Pass the linting checks run by mypy,cd fibonaccicreator && poetry run task mypy,false,b6vKWo8XetQV73efl+7ZIA==,"fibonaccicreator/fibonacci.py:13: error: Name ""List"" is not defined	     fibonaccicreator/fibonacci.py:13: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import List"")	     fibonaccicreator/fibonacci.py:23: error: Name ""Tuple"" is not defined	     fibonaccicreator/fibonacci.py:23: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Tuple"")	     fibonaccicreator/fibonacci.py:35: error: Name ""Tuple"" is not defined	     fibonaccicreator/fibonacci.py:35: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Tuple"")	     fibonaccicreator/fibonacci.py:50: error: Name ""List"" is not defined	     fibonaccicreator/fibonacci.py:50: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import List"")	     Found 4 errors in 1 file (checked 3 source files)",lint
Run the fibonaccicreator program and ensure that it does not crash with iterative list,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativelist,false,b6vKWo8XetQV73efl+7ZIA==,"Traceback (most recent call last):	       File ""<string>"", line 1, in <module>	       File ""/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/importlib/__init__.py"", line 126, in import_module	         return _bootstrap._gcd_import(name[level:], package, level)	       File ""<frozen importlib._bootstrap>"", line 1050, in _gcd_import	       File ""<frozen importlib._bootstrap>"", line 1027, in _find_and_load	       File ""<frozen importlib._bootstrap>"", line 1006, in _find_and_load_unlocked	       File ""<frozen importlib._bootstrap>"", line 688, in _load_unlocked	       File ""<frozen importlib._bootstrap_external>"", line 883, in exec_module	       File ""<frozen importlib._bootstrap>"", line 241, in _call_with_frames_removed	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py"", line 22, in <module>	         from fibonaccicreator import fibonacci	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py"", line 13, in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     NameError: name 'List' is not defined. Did you mean: 'list'?",
Pass the import checks run by isort,cd fibonaccicreator && poetry run task isort,false,33x/IdA+veSNbwwV+sZAeg==,ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py Imports are incorrectly sorted and/or formatted.	     ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/tests/test_fibonacci.py Imports are incorrectly sorted and/or formatted.,lint
Run the fibonaccicreator program and ensure that it does not crash with iterative list,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativelist,true,Rc9Hc3YcbAwPRaxiorY2XA==,,
Pass the linting checks run by flake8,cd fibonaccicreator && poetry run task flake8,true,Rc9Hc3YcbAwPRaxiorY2XA==,,lint
Pass the source code formatting checks run by black,cd fibonaccicreator && poetry run task black,true,Rc9Hc3YcbAwPRaxiorY2XA==,,lint
Pass the linting checks run by mypy,cd contactsearcher && poetry run task mypy,false,cZdEgIWQoeIofDdrYH/Afg==,"contactsearcher/main.py:8: error: Name ""cli"" is not defined	     contactsearcher/main.py:10: error: Name ""typer"" is not defined	     contactsearcher/main.py:11: error: Name ""Optional"" is not defined	     contactsearcher/main.py:11: note: Did you forget to import it from ""typing""? (Suggestion: ""from typing import Optional"")	     contactsearcher/main.py:11: error: Name ""Path"" is not defined	     contactsearcher/main.py:11: error: Name ""typer"" is not defined	     contactsearcher/main.py:19: error: Name ""typer"" is not defined	     contactsearcher/main.py:20: error: Name ""typer"" is not defined	     contactsearcher/main.py:25: error: Name ""typer"" is not defined	     contactsearcher/main.py:30: error: Name ""typer"" is not defined	     contactsearcher/main.py:32: error: Name ""typer"" is not defined	     contactsearcher/main.py:33: error: Name ""typer"" is not defined	     Found 11 errors in 1 file (checked 5 source files)",
Run the fibonaccicreator program and ensure that it does not crash with iterative list,cd fibonaccicreator && poetry run fibonaccicreator --number 1000 --approach iterativelist,false,wC69ew7NwVXVb1dj2hr23w==,"Traceback (most recent call last):	       File ""<string>"", line 1, in <module>	       File ""/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/importlib/__init__.py"", line 126, in import_module	         return _bootstrap._gcd_import(name[level:], package, level)	       File ""<frozen importlib._bootstrap>"", line 1050, in _gcd_import	       File ""<frozen importlib._bootstrap>"", line 1027, in _find_and_load	       File ""<frozen importlib._bootstrap>"", line 1006, in _find_and_load_unlocked	       File ""<frozen importlib._bootstrap>"", line 688, in _load_unlocked	       File ""<frozen importlib._bootstrap_external>"", line 883, in exec_module	       File ""<frozen importlib._bootstrap>"", line 241, in _call_with_frames_removed	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py"", line 13, in <module>	         from fibonaccicreator import fibonacci	       File ""/home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/fibonacci.py"", line 13, in <module>	         def fibonacci_recursivelist(number: int) -> List[int]:	     NameError: name 'List' is not defined. Did you mean: 'list'?",
Pass the linting checks run by pydocstyle,cd fibonaccicreator && poetry run task pydocstyle,true,p5K7+j/WnpdDSeiVoM6iFA==,,lint
Pass the import checks run by isort,cd fibonaccicreator && poetry run task isort,false,b6vKWo8XetQV73efl+7ZIA==,ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/fibonaccicreator/main.py Imports are incorrectly sorted and/or formatted.	     ERROR: /home/runner/work/fibonacci-algorithms-Yanqiao4396/fibonacci-algorithms-Yanqiao4396/fibonaccicreator/tests/test_fibonacci.py Imports are incorrectly sorted and/or formatted.,lint
